charts/portworx/templates/portworx-monitor.yaml

```diff
@@ -466,17 +459,17 @@ metadata:
   labels:
     app: grafana
 spec:
+  replicas: 1
   selector:
     matchLabels:
       app: grafana
-  replicas: 1
   template:
     metadata:
       labels:
         app: grafana
     spec:
       containers:
-        - image: grafana/grafana:5.0.4
+        - image: grafana/grafana:5.3.3
```

Comment:
we should bump this further. Px-Central installs 7.1.3 i believe @sharma-tapas ?
Let's give this a test with 7.1.3 and re-update the chart if it works. i know i have seen some security issues  with this version

Reply:
On-prem-ui is using the old version see here: [https://github.com/portworx/pxcentral-onprem-ui/blob/cc6ad56f2ca340618ee2eb428d8dbe19e5b10d00/dockerGroup/deployments/monitoring/grafana-deployment.yaml#L6064](https://github.com/portworx/pxcentral-onprem-ui/blob/cc6ad56f2ca340618ee2eb428d8dbe19e5b10d00/dockerGroup/deployments/monitoring/grafana-deployment.yaml#L6064 )
We should update both to use the latest 7.1.3, let me talk to Binu in case of PX-Backup monitoring I will confirm with @tusharraut1994 and update.

Reply:
I did a quick test with grafana 7.1.3 and all dashboards look good.

Reply:
@pault84 I just updated image to 7.1.3. Can this PR be merged now?